glvis.cpp

```diff
@@ -157,6 +154,12 @@ bool GLVisInitVis(int field_type)
             vss->SetGridFunction(*grid_f);
          }
          if (field_type == 2)
+         {
+            // Use the 'bone' palette when visualizing a 2D mesh only
+            paletteSet(4);
```

Comment:
Can this go into the next if statement instead? (line 164)

Reply:
Yes, done in https://github.com/GLVis/glvis/pull/146/commits/771f3ad7f2b44988c5421352c3e9653e1b3848c0 :)